### DISCUSSION TITLE
Someone know how to make a setup.exe, for a windows game

### DISCUSSION THREAD

**Discussion by LidesSam** [HaxeFlixel/flixel#2449] [Q&A] [answered]
I have a small demo game, and want to create a installer(setup.exe) for it, to publish in Itch.io. But i don't find any material on how to make i . I program de game in Flash developer.
Someone know how, or were i can learn it?.

**Comment by loudoweb** [comment, marked as answer]
If you really want an installer, try one of these and tell which worked for you, it could help someone else:
https://helpdeskgeek.com/free-tools-review/4-tools-to-create-windows-installer-packages/

**Comment by Geokureli** [comment]
as far I know people just zip up the contents of the exported `bin` folder and upload that to itch. [like this](https://ninja-muffin24.itch.io/funkin)

Itch has a client app that downloads these to the a specified directory, as well.

**Reply by LidesSam** [reply]
Thank I have the client app, it was my first try, but for some reason my computer don't run the client well. and I gonna need the installer later, for another sites, the zip don't gonna work there, is unlike someone not know own to use a zip in itch.io but outside it is another history
